packages/contracts-bedrock/test/L2/L2ToL2CrossDomainMessenger.t.sol

```diff
@@ -0,0 +1,175 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+// Testing utilities
+import { Bridge_Initializer } from "test/setup/Bridge_Initializer.sol";
+import { Reverter, ConfigurableCaller } from "test/mocks/Callers.sol";
+import { EIP1967Helper } from "test/mocks/EIP1967Helper.sol";
+import { ICrossL2Inbox } from "src/L2/ICrossL2Inbox.sol";
+
+// Libraries
+import { Hashing } from "src/libraries/Hashing.sol";
+import { Encoding } from "src/libraries/Encoding.sol";
+import { Types } from "src/libraries/Types.sol";
+
+// Target contract dependencies
+import { L2ToL1MessagePasser } from "src/L2/L2ToL1MessagePasser.sol";
+import { AddressAliasHelper } from "src/vendor/AddressAliasHelper.sol";
+
+contract L2ToL2CrossDomainMessengerTest is Bridge_Initializer {
+    address origin = address(l2ToL2CrossDomainMessenger);
+    uint256 destination = block.chainid;
```

Comment:
I could be better to fuzz in cases where these are used and in cases where fuzzing doesn't make sense, instead use local variables. I think this helps to keep the tests cleaner and more portable